news/2020-04-23-osu-mapping-olympiad-6-results.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "osu! Mapping Olympiad #6 Results (osu!catch)"
+date: 2020-04-23 17:00:00 +0000
+---
+
+Apples, oranges, bananas and... waffles? The results for the second osu!catch Mapping Olympiad contest are in! See who came out on top to join the coveted ranks of the Elite Mappers!
+
+[![](https://osu.ppy.sh/wiki/shared/news/banners/osu_mapping_olympiad.jpg)](https://osu.ppy.sh/community/contests/90)
```

Comment:
banners don't support links anymore, maybe link "the second osu!catch Mapping Olympiad contest" in the paragraph above this instead

Reply:
decided to put at the start of the paragraph below this instead, since I don't really recall seeing links in headers/blurbs